runbotics/runbotics-scheduler/src/scheduler-database/tenant/tenant.controller.ts

```diff
@@ -0,0 +1,143 @@
+import {
+    BadRequestException, Body, Controller, Get,
+    NotFoundException, Param, ParseUUIDPipe, Patch,
+    Post, UseInterceptors
+} from '@nestjs/common';
+import { FeatureKey, Tenant } from 'runbotics-common';
+
+import { FeatureKeys } from '#/auth/featureKey.decorator';
+import { ZodValidationPipe } from '#/utils/pipes/zod-validation.pipe';
+import { User } from '#/utils/decorators/user.decorator';
+import { UserEntity } from '#/database/user/user.entity';
+import { Logger } from '#/utils/logger';
+import { TenantInterceptor } from '#/utils/interceptors/tenant.interceptor';
+
+import { TenantService } from './tenant.service';
+import { CreateTenantDto, createTenantSchema } from './dto/create-tenant.dto';
+import { UpdateTenantDto, updateTenantSchema } from './dto/update-tenant.dto';
+
+@Controller('/api/scheduler/tenants')
+export class TenantController {
+    private readonly logger = new Logger(TenantController.name);
+
+    constructor(
+        private readonly tenantService: TenantService
+    ) { }
+
+    @Get('/:tenantId/me')
+    @UseInterceptors(TenantInterceptor)
+    @FeatureKeys(FeatureKey.TENANT_READ)
+    async getTenantByUser(@User('tenantId') id: Tenant['id']) {
```

Comment:
you should make types for specific user key, based on request param you are expecting